instrumentation/aws-sdk/aws-sdk-1.11/library-autoconfigure/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/autoconfigure/TracingRequestHandler.java

```diff
@@ -21,8 +21,8 @@ public class TracingRequestHandler extends RequestHandler2 {
   private static final RequestHandler2 DELEGATE =
       AwsSdkTelemetry.builder(GlobalOpenTelemetry.get())
           .setCaptureExperimentalSpanAttributes(
-              Config.get()
-                  .getBoolean("otel.instrumentation.aws-sdk.experimental-span-attributes", false))
+              ConfigPropertiesUtil.getBoolean(
+                  "otel.instrumentation.aws-sdk.experimental-span-attributes", false))
```

Comment:
`library-autoconfigure` modules have to keep on using env var/system prop configuration, as it's their only way of configuring themselves.